Makefile

```diff
@@ -53,7 +53,7 @@ clean-test:
 	rm -f test-results.xml
 
 lint:
-	flake8 .
+	pre-commit run --all-files
```

Comment:
Add ` --show-diff-on-failure` here so that when it does fail on something the pre-commit modifies it will print out the diffs making it easier to spot what the cause of the faillure was.

Reply:
done